spec/inertia/request_spec.rb

```diff
@@ -154,4 +154,12 @@
       end
     end
   end
+
+  describe 'a non existent route' do
+    it 'raises a 404 exception' do
```

Comment:
Arguably this is just testing Rails instead of InertiaRails, but I wrote this to reproduce a possible bug that @skryukov found in the middleware, so I'm leaving it in.